.changeset/tiny-kids-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`HDS::Reveal` - Added support for `aria-describedby` attribute on toggle.
```

Comment:
[todo] you have to use the new format for the changelog entries: https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3243114706/Changelog+authoring+best+practices#Component-formatting-for-npm-packages